setup.py

```diff
@@ -29,6 +29,7 @@ def get_install_requires() -> List[str]:
 
     requirements = [
         "alembic",
+        "click==7.1.2",  # Remove after click 8.0.0 import issue fixed
```

Comment:
`click` is not required by optuna, so we don't want to install it by default.
`click` was only required when testing the integration modules and examples, so how about install it in the GitHub Actions workflows? In #2474, we installed `cython` in the tests workflow.

Reply:
Thanks for the clarification. I agree with @toshihikoyanase. Let me dismiss my previous review. Sorry for the confusion.